source/main.ts

```diff
@@ -47,6 +48,9 @@ if (!app.requestSingleInstanceLock()) {
 
 // If we reach this point, we are now booting the first instance of Zettlr.
 
+// Use a proxy if one has been configured
+bootstrap()
```

Comment:
If not necessary to do so before the `appReady` event, please move this call out of the main file and into the boot cycle. If this needs to be called first thing even before the boot cycle, please add a comment why.